### ISSUE TITLE
Improve feature admin functions

### ISSUE THREAD

**Issue by agjohnson** [rtfd/readthedocs.org#3216]
- [x] Make inline for project -> feature relationship
- [x] Make projects field raw_id_field or something readonly? The # of projects might cause problems here
- [ ] Add admin function for adding sample size of projects to feature. Get X random projects from the database. Make sure query isn't awful on database

**Comment by humitos** [comment]
There is only one action missing here.

I imagine this flow for that action:

1. go to `/admin/projects/feature/`
1. select the feature you want to add to projects
1. select the action `Assign to a sampled random projects` (or similar)
1. present a new page with a form
1. enter the number of projects and save
1. show another page with all the projects affected

**Comment by dojutsu-user** [comment]
@humitos 
I am a little confused with the term `sampled random projects`. Can I get a little description about what are those projects.
I am willing to work on this.

**Comment by dojutsu-user** [comment]
I will make a PR for this in a few days.
But what will be the advantage of "picking N random projects and assigning them a particular feature".
Is it for testing those features?

**Comment by humitos** [comment]
Yes. Sometime we introduce a feature and we don't want to apply it to all the projects because we are not 100% sure if it will work for _all of them_. So, in those cases we want to pick randomly an amount of projects and apply this new feature.

After some time, if we didn't notice anything strange or wrong with these projects, we can apply the feature to another random sample of projects or apply to all of them. But we want to have that flexibility.

**Comment by dojutsu-user** [comment]
@humitos 
Thank you for the explanation.

**Comment by dojutsu-user** [comment]
This is still valid I think.

**Comment by dojutsu-user** [comment]
Still valid.
